packages/blazor-workspace/Tests/NimbleBlazor.Tests.Acceptance.Shared/NimbleBlazor.Tests.Acceptance.Shared.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <NoWarn>CA1716</NoWarn>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <AdditionalFiles Include="..\..\CodeAnalysisDictionary.xml" Link="CodeAnalysisDictionary.xml" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="6.0.29" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="7.0.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.11.0" />
+    <PackageReference Include="Microsoft.Playwright" Version="[1.42.0]" />
+    <PackageReference Include="NI.CSharp.Analyzers" Version="[2.0.21]" />
+    <PackageReference Include="System.ComponentModel" Version="4.3.0" />
+    <PackageReference Include="xunit" Version="2.8.0" />
+    <PackageReference Include="xunit.extensibility.execution" Version="2.8.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.8.0">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+  </ItemGroup>
+
```

Comment:
Make sure to set DisableImplicitNuGetFallbackFolder,

```
<PropertyGroup>
  <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
</PropertyGroup>
```

Reply:
@atmgrifter00 @msmithNI is there other configuration we need to add per project? We should make sure that is captured in contributing docs for when additional projects are added.

Reply:
Good catch @rajsite .

I think the per-project configuration is something like:
- .NET version: should match other existing projects generally (`net6.0` for now)
- `<RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>` and `<DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>`
- Reference to `NI.CSharp.Analyzers` (version spec should match other existing projects)

@m-akinc if you want to take a stab at putting that in the blazor-workspace contributing docs (in a new section for 'adding new projects') feel free, otherwise we (owners) can follow up on that point.

Reply:
Added to the contributing doc.